plugins/ocp/ocp-nvme.h

```diff
@@ -14,6 +14,8 @@
 #define OCP_PLUGIN_VERSION   "2.9.3"
 #include "cmd.h"
 
+#define ENABLE_IEEE1667_SILO "enable IEEE1667 silo"
```

Comment:
I would use the string directly in the command list macro. Just to keep it consistent with the rest.